data/src/main/kotlin/com/alfresco/content/data/AnalyticsRepository.kt

```diff
@@ -111,11 +110,40 @@ enum class EventName(val value: String) {
     OpenWith("Event_openWith")
 }
 
+/**
+ * Marked as PageView enum
+ */
+enum class PageView(val value: String) {
+    Recent("PageView_Recent"),
+    Favorites("PageView_Favorites"),
+    Offline("PageView_Offline"),
+    Browse("PageView_Browse"),
+    PersonalFiles("PageView_PersonalFiles"),
+    FolderName("PageView_FolderName"),
+    MyLibraries("PageView_MyLibraries"),
+    Shared("PageView_Shared"),
+    Trash("PageView_Trash"),
+    Search("PageView_Search"),
+    ShareExtension("PageView_ShareExtension"),
+    Transfers("PageView_Transfers"),
+    None("none")
+}
+
+/**
+ * Marked as APIEvent enum
+ */
+enum class APIEvent(val value: String) {
+    SyncingStatus("Event_API_SyncingStatus"),
```

Comment:
As discussed, no need of this sync event

Reply:
I'll remove the same in the next ticket(MOBILEAPPS-1239)